### ISSUE TITLE
[Low]: Chrome+JAWS: Screen Reader: Change of language is not identified by Screen Reader.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9634]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Login to the Expensify app with the required credentials.
3. Click on User Profile and a Settings pop-up will appear.
4. Select Preferences from the pop-up and the Preferences pop-up will appear.
5. From the language dropdown select "Espanol" language option.
6. Click on Back button and select Payments.
7. Click on Add a Payment method and a dropdown list will appear.
8. Click on Add a debit card option from it.
9.  Navigate to the "Guardar" button with the Screen reader and observe the reading of the screen reader.

## Expected Result:
Screen Reader should pronounce the "Guardar" button in Spanish accent

## Actual Result:
Screen Reader is pronouncing the "Guardar" button in a normal English accent.

## Workaround:
Yes

## Area issue was found in:
Add a debit card.

## Failed WCAG checkpoints
3.1.2

## User impact:
Screen reader and cognitive users were not able to recognize the change of language.

## Suggested resolution:
Provide "lang=ES" attribute in the HTML whenever the change of language is done.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*v1.1.79-3*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*
